### ISSUE TITLE
Improve label for investment chart

### ISSUE THREAD

**Issue by vgeorge** [global-electrification-platform/explorer#238]
Comment by Chiara:

> ‘Additional” investment requirement for 2030, as they are additional to 2025 (semantics on the website)

Is it sufficient to add a comment in the popover like we did for the population chart?

## Actual Investment Chart

![Screenshot 2019-10-08 15 00 11](https://user-images.githubusercontent.com/329694/66402409-b4636300-e9dc-11e9-8621-f7f85da1743a.png)

## Population Chart Popover

![Screenshot 2019-10-08 15 00 17](https://user-images.githubusercontent.com/329694/66402445-c218e880-e9dc-11e9-970d-de2293c02422.png)

cc @akorkovelos @olafveerman

**Comment by nicolinalindblad** [comment]
What we need are the accumulated numbers for 2030. Meaning when the 2030 scenario is shown, the graphs should indicate the required investments etc. for the entire model period (2018-2030), and not only 2025-2030 as it appears now. This requires some work I suppose on the backend too @akorkovelos

**Comment by nicolinalindblad** [comment]
I just wanted to follow up and see how you are doing on this/ if you need any input?

**Comment by olafveerman** [comment]
@nicolinalindblad 
We made the necessary changes to the backend to show them in the Explorer. Once a new version of the backend is released, this will be unblocked.

**Comment by vgeorge** [comment]
This will be fixed when https://github.com/global-electrification-platform/data-service/pull/94 is merged and deployed.

**Comment by nicolinalindblad** [comment]
Thank you! @vgeorge and @olafveerman

**Comment by vgeorge** [comment]
@nicolinalindblad the live version is displaying accumulated investment values from start year.

Example:

- https://electrifynow.energydata.info/explore/bj-1?year=2025&scenario=0_0_0_0_0_0&filters=r5_1242249%7Cr0_254%7C1_3_5_6_7%7Cr0_80%7Cr0_25%7Cr23_2456

- https://electrifynow.energydata.info/explore/bj-1?year=2030&scenario=0_0_0_0_0_0&filters=r5_1242249%7Cr0_254%7C1_3_5_6_7%7Cr0_80%7Cr0_25%7Cr23_2456

**Comment by AndreasSahlberg** [comment]
Dear all, 
It seems like all summary charts are correct with the exception of Investment 2030. 

It seems like it currently is:
`INVESTMENT REQUIRED = (InvestmentCost2025 + InvestmentCost2030) x ElecStatusIn2030`

But it should be:
`INVESTMENT REQUIRED = (InvestmentCost2025 x ElecStatusIn2025) + (InvestmentCost2030 x ElecStatusIn2030)`

To clarify, exactly the same calculations that were made for 2025 should be replicated for 2030 and these should be summed to get the cumulative investments. As it is now, the total investments are a bit too high (taking into account the investment cost in 2025 also for settlements that were actually not electrified then), and also does not consider the fact that we might have a change of technologies between 2025 and 2030.

**Comment by vgeorge** [comment]
Hi @AndreasSahlberg, thanks for catching this. We are working on a fix at https://github.com/global-electrification-platform/data-service/pull/98. We will let you know when it is done.

**Comment by nicolinalindblad** [comment]
Andreas, how are the different costs for respective technology being
accounted for here?



Den tis 28 jan. 2020 kl 08:23 skrev Vitor George <notifications@github.com>:

**Comment by vgeorge** [comment]
@AndreasSahlberg a fix on investment cost was pushed to the API, please review if it is following your description.

**Comment by AndreasSahlberg** [comment]
Dear @vgeorge ,

Thank you for this! I have reviewed some scenarios for Angola (ao), Benin (bj) and Botswana (bw) on the platform. 

It seems to me like the following scenarios are correct after the changes:
ao-1-0-2-0-0-0-0
ao-1-0-1-0-0-0-0
bj-1-1-0-0-0-0-0
bj-1-0-1-0-0-0-0
bj-1-0-2-0-0-0-0
bw-1-1-0-0-0-0-0
bw-1-0-1-0-0-0-0
bw-1-0-2-0-0-0-0

However, the following are correct for 2025 but not for 2030 (where the investment cost is too high):
ao-1-0-0-0-0-0-0
ao-1-1-0-0-0-0-0
bj-1-0-0-0-0-0-0
bw-1-0-0-0-0-0-0

Not sure if the changes have not been implemented for all scenarios yet, or if there is another issue to look into.
